modules/backing-up-red-hat-quay.adoc

```diff
@@ -0,0 +1,320 @@
+  namespace: <quay-namespace>
+data:
+  CONFIG_EDITOR_PW: <redacted>
+  DATABASE_SECRET_KEY: <redacted>
+  DB_ROOT_PW: <redacted>
+  DB_URI: <redacted>
+  SECRET_KEY: <redacted>
+  SECURITY_SCANNER_V4_PSK: <redacted>
+----
++
+All information under the `data` property should remain the same.
+
+. Backup the current Quay configuration. This configuration lives in a secret pointed by the `QuayRegistry` custom resource and you can find the name by running the following command:
++
+[source,terminal]
+----
+$ oc get quayregistry <quay-registry-name> -n <quay-registry-namespace> -o json |jq -r .spec.configBundleSecret
+----
+
+. After obtaining the configuration bundle name, move its content into a new file:
++
+[source,terminal]
+----
+$ oc get secret -n <quay-registry-namespace> <config-bundle-secret-name> -o yaml > config-bundle.yaml
+----
+
+. Scale down the Quay component and edit the QuayRegistry custom resource by unmanaging the `horizontalpodautoscaler` component:
++
+[source,terminal]
+----
+$ oc edit quayregistry <quay-registry-name> -n <quay-registry-namespace>
+----
++
+[source,yaml]
+----
+  apiVersion: quay.redhat.com/v1
+  kind: QuayRegistry
+  metadata:
+    name: registry
+    namespace: ns
+  spec:
+    components:
+      …
+      - kind: horizontalpodautoscaler
+        managed: false
+      - kind: quay
+        managed: true
+        overrides:
+          replicas: 0
+      …
+----
++
+[NOTE]
+====
+If you are using a version older than 3.7.0, then you must manually scale down the Operator and manually scale down the Quay deployment. For example, you can run `$ oc scale --replicas=0 deployment quay-operator.v3.6.2 -n openshift-operators` to scale down the Operator, and `$ oc scale --replicas=0 deployment QUAY_MAIN_APP_DEPLOYMENT` to scale down the main application deployment.
+====
+
+. Wait for the `registry-quay-app` pods to disappear. You can check their status by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n <quay-namespace>
+----
++
+Example output:
++
+[source,terminal]
+----
+registry-quay-config-editor-77847fc4f5-nsbbv   1/1     Running            0          9m1s
+registry-quay-database-66969cd859-n2ssm        1/1     Running            0          6d1h
+registry-quay-mirror-758fc68ff7-5wxlp          1/1     Running            0          8m29s
+registry-quay-mirror-758fc68ff7-lbl82          1/1     Running            0          8m29s
+registry-quay-redis-7cc5f6c977-956g8           1/1     Running            0          5d21h
+----
+
+. Identify your Quay database pod:
++
+----
+$ oc get pods -n <quay-namespace> | grep quay-database
+----
++
+Example output:
++
+----
+example-registry-quay-database-66969cd859-n2ssm        1/1     Running            0          6d1h
+----
+
+. Create and download a backup database:
++
+----
+$ oc exec example-registry-quay-database-66969cd859-n2ssm -- /usr/bin/pg_dump -C example-registry-quay-database > backup.sql
+----
```

Comment:
It is better to get specific quay database name from configration file 

Maybe we can refine line 128 to line 144 as
```
* Get the quay postgrepsql pod name
----
$ oc get pod -l quay-component=postgres -n <quay-namespace> -o jsonpath='{.items[0].metadata.name}'
quayregistry-quay-database-59f54bb7-58xs7 
----

* Get the quay database name
----
$ oc -n <quay-namespace> rsh $(oc get pod -l app=quay -o NAME -n <quay-namespace>|head -n 1) cat /conf/stack/config.yaml|awk -F"/" '/^DB_URI/ {print $4}'
quayregistry-quay-database
----

* Download a backup database:
----
oc exec quayregistry-quay-database-59f54bb7-58xs7 -- /usr/bin/pg_dump -C quayregistry-quay-database  > backup.sql
----
```